File: sdk/client-ts/src/___PLAYGROUND___/relayer/get-relayer.ts
import { begin } from '../helpers';

export const getRelayer = async () => {
  const context = await begin();

  console.log('Getting relayer info...');
  const relayerInfo = await context.client.relayer.get(context.relayerInfo.id);
  console.log('Relayer info:', relayerInfo);

  await context.end();
};

getRelayer().then(() => console.log('get-relayer done'));

File: sdk/python/src/__PLAYGROUND__/relayer/get-relayer.py
import asyncio

from __PLAYGROUND__.helper import begin, end


async def getRelayer():
    rrelayer_node = None
    try:
        client, _, info, rrelayer_node = await begin()

        print("Getting relayer info...")
        relayerInfo = await client.relayer.get(info["id"])
        print("Relayer info:", relayerInfo)
    except Exception as e:
        print("getRelayer failed:", e)
    finally:
        if rrelayer_node is not None:
            end(rrelayer_node)


if __name__ == "__main__":
    asyncio.run(getRelayer())
    print("get-relayer done")
